src/Props/speakers/cards.ts

```diff
@@ -0,0 +1,55 @@
+import { CardProps } from "Props/props"
+import images from "./images"
+
+const speakerCards: CardProps[] = [
+  {
+    name: "Antonio Xu Liu",
+    title: "Senior Strategy Program Manager\n@ National Grid",
+    imgSrc: images.seaOtter,
+    LinkedInLink: "",
```

Comment:
If we need to include more information about the speakers, I can make another interface for this. Right now I'm using the same interface as our team cards, since they are almost the same.